agent/rpm/pbench-agent.spec.j2

```diff
@@ -8,16 +8,22 @@ URL:            https://github.com/distributed-system-analysis/pbench
 Source0:        pbench-agent-%{version}.tar.gz
 Buildarch:      noarch
 
-%if 0%{?rhel} < 9
-Requires:  ansible
-%else
+# EPEL provides ansible (a curated set of roles with a dependency on ansible-core)
+# on RHEL 8 and RHEL9. Only CentOS-Stream seems to not have an ansible package
+# available.
+
+%if 0%{?centos} >= 8
 Requires:  ansible-core
+%else
+Requires:  ansible
 %endif
 
-%if 0%{?rhel} == 7
+
+%if 0%{?rhel} == 7 
 Requires:  python3, python3-pip
 %endif
 
+# this applies to CentOS-Stream-8 as well
 %if 0%{?rhel} == 8
 Requires:  python36, python3-pip
```

Comment:
I'm not quite sure what this comment is trying to tell me...is it saying that, on CentOS Stream 8, `{rhel}` will be `8`, so that this requirement will be added?  If so, I'm glad you added the comment, because that's rather counter-intuitive...and I'd be inclined to make the comment a little more expressive.  Perhaps, "This condition will be true on CentOS-Stream-8 as well" (or even `Note:  {?rhel} will evaluate to '8' on CentOS-Stream-8`).

Likewise for line 35.

Reply:
That's exactly what I'm saying.  I'll change the comments.